src/plugins/intel_npu/src/plugin/src/metrics.cpp

```diff
@@ -91,6 +91,25 @@ IDevice::Uuid Metrics::GetDeviceUuid(const std::string& specifiedDeviceName) con
     return IDevice::Uuid{};
 }
 
+std::map<std::string, double> Metrics::GetUtilization(const std::string& specifiedDeviceName) const {
+    std::string uuidStr;
+    double utilization = 0.0;
+    try {
```

Comment:
please revert this and just return a default value based on the return error code.